esy.json

```diff
@@ -4,18 +4,20 @@
   "license": "MIT",
   "version": "3.5.0",
   "dependencies": {
```

Comment:
i used esy add and it shuffled things around (i have version of esy running off master)

Reply:
Yeah, that's fine.